README.md

```diff
@@ -146,3 +146,20 @@ void update()
 
 <br/>
 
+
+
+---
+
+# Linux
+
+## Pre-requisites
+
+```bash
+# Arch Linux
+sudo pacman -S meson sdl12-compat sdl2_ttf base-devel
```

Comment:
@Avicted Did you forget to include `sdl2` package or am I missing something?

Reply:
mb, just checked the dependencies for `sdl12-compat`, and `sdl2` is listed among them.